prow/lib.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
```

Comment:
in the future we should probably put this in common repo. Lets get the prow update done first though in case it changes

Reply:
i was thinking of doing that before proceeding with this work but had same reaction that should go in after the prow update because it will probqably end in a rewrite of this library.